src/xo-app/menu/index.js

```diff
@@ -134,13 +142,14 @@ export default class Menu extends Component {
 }
 
 const MenuLinkItem = props => {
-  const { item } = props
-  const { to, icon, label, subMenu } = item
+  const { item, collapsed } = props
+  const { to, icon, label, subMenu, pill } = item
 
   return <li className='nav-item xo-menu-item'>
-    <Link activeClassName='active' className='nav-link' to={to}>
-      <Icon icon={`menu-${icon}`} size='lg' fixedWidth />
-      <span className={styles.hiddenCollapsed}>{' '}{_(label)}</span>
+    <Link activeClassName='active' className={collapsed ? 'text-xs-center nav-link' : 'nav-link'} to={to}>
```

Comment:
Avoid passing a `collapsed` prop, simply add a `.centerCollapsed` style in `menu/index.scss`.